Chapter12/recipe-03/cxx-example/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+# set minimum cmake version
+cmake_minimum_required(VERSION 3.1 FATAL_ERROR)
+
+# project name and language
+project(recipe-03 CXX)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_EXTENSIONS OFF)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+add_library(sum_integers sum_integers.cpp)
+
+add_executable(sum_up main.cpp)
+target_link_libraries(sum_up sum_integers)
+
+# we will use the network to fetch Google Test sources
+# make it possible to disable unit tests when not on network
+option(ENABLE_UNIT_TESTS "Enable unit tests" ON)
+message(STATUS "Enable testing: ${ENABLE_UNIT_TESTS}")
+
+if(ENABLE_UNIT_TESTS)
+    include(ExternalProject)
+
+    ExternalProject_Add(
+        gtest
+        PREFIX
+          "${PROJECT_BINARY_DIR}/gtest"
+        GIT_REPOSITORY
+          https://github.com/google/googletest.git
+        GIT_TAG
+          release-1.8.0
+        INSTALL_COMMAND
```

Comment:
I would set the install prefix to something user-writable instead of having a no-op as `INSTALL_COMMAND`

Reply:
I tried but failed but I need to try again since the "true" is also not portable.